src/containers/TaskRun/TaskRun.js

```diff
@@ -32,9 +32,11 @@ import TaskTree from '../../components/TaskTree';
 import { getStatus, stepsStatus, taskRunStep } from '../../utils';
 
 import '../../components/Run/Run.scss';
-import { fetchTask } from '../../actions/tasks';
+import { fetchTask, fetchTaskByType } from '../../actions/tasks';
 import { fetchTaskRun } from '../../actions/taskRuns';
 
+const taskTypeKeys = { ClusterTask: 'clustertasks', Task: 'tasks' };
```

Comment:
This is OK here for now but may need to move if we need to process / display ClusterTasks elsewhere